### PR TITLE
initial contribution

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

**Comment by Ladicek** [comment]
I asked @michalszynkiewicz for review because he's the PR reviewer this week :-), but perhaps there would be a better reviewer. I know @manovotn already looked at this codebase.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+    <properties>
+        <jdk.min.version>9</jdk.min.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.jboss.modules</groupId>
+            <artifactId>jboss-modules</artifactId>
+            <version>1.8.7.Final</version>
```
Comment (manovotn) [inline pom.xml:29]: I suppose the reason to have this version (latest is 1.9.0) is to mirror that of WFLY which is a base for TT?
Or can you update jboss-modules in TT independently?

Comment (Ladicek) [reply pom.xml:29]: Exactly, it should match what is in Thorntail. Let me add a comment.

Comment (Ladicek) [reply pom.xml:29]: Ah, one more. I'm pretty sure we _can_ update JBoss Modules independently of WildFly, but they probably shouldn't be too different :-)

**Review by manovotn — APPROVED** [review]

**Pull request merged**